etc/helm/pachyderm/templates/pachd/rbac/role.yaml

```diff
@@ -47,4 +47,15 @@ rules:
   - update
   - delete
   - deletecollection
+- apiGroups:
+  - "apps"
+  - "extensions"
+  resources:
+  - deployments
+  - deployments/scale
+  verbs:
+  - get
+  - list
+  - patch
```

Comment:
just a reminder that we should be able to remove these perms in case we decide not to modify the deployments

Reply:
Still need them for the deployment rolling portion.